assets/sass/local.scss

```diff
@@ -2,6 +2,9 @@
 .location-bar .location-bar__location a:visited {
   color: $govuk-link-colour;
 }
+.location-bar .location-bar__location a {
+  font-weight: 400;
```

Comment:
Rather than specifying a number for the weight, I'd be tempted to either:
* use `$govuk-font-weight-regular` here (see [Sass API Reference](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-font-weight-regular)); or,
* instead add the `govuk-!-font-weight-regular` class in the template (see [Font override classes](https://design-system.service.gov.uk/styles/typography/#font-override-classes))